content/docs/reference/spec/migration/buildpack-api-0.5-0.6.md

```diff
@@ -0,0 +1,39 @@
++++
+title="Buildpack API 0.5 -> 0.6"
++++
+
+<!--more-->
+
+This guide is most relevant to buildpack authors.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.6) for buildpack API 0.6 for the full list of changes and further details.
+
+### Opt-in layer caching
+
+Buildpacks must now explicitly opt-in to layer re-use by writing `launch`, `build`, or `cache` keys to a new `[types]` table in `<layers>/<layer>.toml` (note that these keys were removed from the top level). If buildpacks do not modify `<layers>/<layer>.toml`, the layer will not be re-used, even if the buildpack in the previous build set any of these keys to `true`.
+
+A Bash buildpack could write something like the following to `<layers>/<layer>.toml` in order to cache a layer:
+
+```bash
+cat >> layer.toml <<EOL
+[types]
+cache = true
+EOL
+```
+
+### Buildpacks contribute default process type
+
+Buildpacks may now contribute the default process type by writing `default = true` in the process type definition in `<layers>/launch.toml`. An individual buildpack may only specify one process type with `default = true`. The lifecycle will choose, from all buildpack-provided process types, the last process type with `default = true` as the buildpack-provided default. A user may override the buildpack-provided default process type by passing `-process-type` to the exporter. Note: to enable buildpacks implementing older Buildpack APIs to work with newer buildpacks, the lifecycle will assume that buildpacks on Buildpack API < 0.6 intended for `web` processes to be the default.
+
```

Comment:
```suggestion
For example - 

```toml
# launch.toml
[[processes]]
type = "<process type>"
command = "<command>"
args = ["<arguments>"]
direct = false
# the default field is new
default = false
```
```